server/views/applications/pages/health-needs/brain-injury.njk

```diff
@@ -3,8 +3,180 @@
 
 {% block questions %}
 
-  <p class="govuk-body">
-    Brain injury page
+  {% set injuryAndNeedsFollowUp %}
+  {{
+      formPageInput(
```

Comment:
the `formPageInput`s should all be `formPageTexArea`s as well, unless you've had a chat with design!

Reply:
👍 thks!